packages/web/pages/[technology]/Header.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from '../../components/Button';
+import { useTechnology } from '../../hooks';
+
+const defaultThumbnail = 'https://rocketfinalchallenge.s3.amazonaws.com/card-image.jpg';
+
+const Header = () => {
+	const { technology } = useTechnology();
+
+	return (
+		<>
+			<MainTitle>{technology.title}</MainTitle>
+
+			<HeaderContainer>
+				<ImageContainer
+					src={technology.thumbnail || defaultThumbnail}
+					alt={technology.title}
+					data-testid="image"
+				/>
+				<DescriptionContainer>
+					<DescriptionTitle>{technology.title}</DescriptionTitle>
+					<DescriptionText>{technology.description}</DescriptionText>
+					<ActionsContainer>
+						<ImplementationCost />
+						<ButtonsContainer>
+							<Button variant="success">Quero Adquirir Essa Tecnologia</Button>
+							<Button variant="info">Quero Suporte Para Essa Tecnologia</Button>
+						</ButtonsContainer>
+					</ActionsContainer>
+				</DescriptionContainer>
+			</HeaderContainer>
+		</>
+	);
+};
+
+export const MainTitle = styled.h1`
+	color: ${({ theme: { colors } }) => colors.secondary};
+	border-bottom: 1px solid ${({ theme: { colors } }) => colors.lightGray4};
+	text-align: start;
+	font-size: 2.8rem;
+	font-weight: 600;
+`;
+
+export const DescriptionTitle = styled.h2`
+	color: ${({ theme: { colors } }) => colors.secondary};
+	text-align: start;
+	font-size: 2.8rem;
+	font-weight: 600;
+
+	@media (max-width: ${({ theme }) => theme.screens.medium}px) {
+		margin-top: 1rem;
+	}
+`;
+
+export const ImageContainer = styled.img`
+	margin: 1rem 1rem 1rem 0;
+	width: 100%;
+	max-width: 450px;
+	height: auto;
+	border-radius: ${({ theme: { metrics } }) => metrics.smallRadius}rem;
+	-webkit-box-shadow: 0.5px 0.5px 10px 0 ${({ theme: { colors } }) => colors.lightGray3};
+	box-shadow: 0.5px 0.5px 10px 0 ${({ theme: { colors } }) => colors.lightGray3};
```

Comment:
Aqui sugiro fazer aquela mesma abordagem lá do `styles` do Tabs, pegando os valores no início do componente  e depois só reutilizando.